stdlib/StdlibModules

```diff
@@ -18,68 +18,71 @@
 # This file lists all standard library modules.
 # It is used in particular to know what to expunge in toplevels.
 
-P ?= stdlib__
+define add_stdlib_prefix
+ $(or $(filter stdlib, $1), $(filter camlinternal%, $1), stdlib__$1)
+endef
```

Comment:
This can be slightly simpler:
```make
# Add stdlib__ prefix to all non-internal modules except stdlib itself
add_stdlib_prefix=$(or $(filter stdlib camlinternal%,$1),stdlib__$1)
```